src/main/kotlin/it/pagopa/wallet/services/WalletService.kt

```diff
@@ -363,6 +363,13 @@ class WalletService(
         return walletRepository.findByUserId(userId.toString()).collectList().map { toWallets(it) }
     }
 
+    fun findWalletAuthData(walletId: UUID): Mono<WalletAuthDataDto> {
```

Comment:
You can use as an input parameter in the service the @ValueObject WalletId

Reply:
done :-)